tests/requirements.txt

```diff
@@ -2,5 +2,4 @@ pytest
 ragger[speculos,ledgerwallet]>=1.11.4
 ecdsa
 simple-rlp
-# to change when ethereum client will be pushed on master
-tests/client/
\ No newline at end of file
+ledger_app_clients.ethereum
```

Comment:
It seems that there are some conflicts between python packages when running tests with the CI.

Reply:
solved